CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+Please follow the [guidance](#adding-to-this-changelog) at the bottom of this file when making changes
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
+and follows a [Backwards Compatability Policy]
+(https://docs.solana.com/developing/backwards-compatibility)
+
+## Unreleased
+### [1.17.0]
+* Changes
+  * Added a changelog!
+* Upgrade Notes
+
+## Released
+### 2023-05-31 - [1.16.0](https://github.com/solana-labs/solana/releases/tag/v1.16.0)
+
+## Adding to this Changelog
+### Audience
+* Entries in this log are intended to be easily understood by contributors,
+consensus validator operators, rpc operators, and dapp developers.
+
+### Noteworthy
+* A change is noteworthy if it:
+  * Adds a feature gate, or
+  * Implements a SIMD, or
+  * Modifies a public API, or
+  * Changes normal validator / rpc run configurations, or
+  * Changes command line arguments, or
+  * Fixes a bug that has received public attention, or
+  * Significantly improves performance, or
+  * Is authored by an external contributor.
+
+### Instructions
+* Update this log in the same pull request that implements the change. If the
+change is spread over several pull requests update this log in the one that
+makes the feature code complete.
+* Add notes to the [Unreleased] section in each branch that you merge to.
+  * Add a description of your change to the Changes section.
+  * Add Upgrade Notes if the change is likely to require:
+    * validator or rpc operators to update their configs, or
```

Comment:
Would be good to develop a standard to highlight this to those that are affected. Could be a marker
* [\<insert affected list here\>] Explanation of the change and how it affects each group

Reply:
I'm inclined to leave this sort of annotation as a step for comms, but I'm open to asking engineers to do it if anyone feels strongly.